cerebrium/advanced-functionality/using-secrets.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Using Secrets"
+---
+
+You may want to use secrets in your model. It's best to store these as secrets.
```

Comment:
This can be worded better:

"You may want to use keys, password or any sensitive information in your application but not have it stored in your code. If this is the case, it would be best to make use of our secrets functionality. Its encrypted at rest etc